### PR TITLE
Implement GET CITY POSTS request

### PR DESCRIPTION
**Pull request by 1jeanpaul1**
## Problem

Need a way for users to see the posts in their given city

## Solution

Add endpoint

## Test
Try the following with different parameters: 
http://localhost:8082/forum/post/v1/cityPosts?cityId=1&page=0&size=3

### CODE REVIEW

File: src/main/java/com/fcgl/madrid/forum/service/CommentService.java [hunk 1]
```diff
@@ -45,13 +54,35 @@ public CommentService(CommentRepository commentRepository) {
...
+
+        boolean success = updatePostCommentCount(commentRequest.getPost(), new Integer(1));
+        if (!success) {
+            logger.warn(new String("UPDATE POST COMMENT COUNT FAILED FOR POST: " + Long.toString(commentRequest.getPost().getId())));
+        }
+        return new ResponseEntity<InternalStatus>(InternalStatus.OK, HttpStatus.OK);
     }
 
+    /**
+     * Adds a count to the number of comments. Follows the "eventually consistent strategy".
```
Comment (1jeanpaul1) [inline src/main/java/com/fcgl/madrid/forum/service/CommentService.java:70]: Okay, we still need to create the process to make commentSize eventually consistent. We can do this is a separate PR. Basically whenever this logger is set, we will eventually want to update the `commentSize` to the actual size.

**Review by fcgl — APPROVED** [review]
Looks good
